src/CommandHandler/Wishlist/ImportWishlistFromCsvHandler.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\CommandHandler\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\ImportWishlistFromCsv;
+use BitBag\SyliusWishlistPlugin\Controller\Action\AddProductVariantToWishlistAction;
+use Gedmo\Exception\UploadableInvalidMimeTypeException;
+use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+final class ImportWishlistFromCsvHandler implements MessageHandlerInterface
+{
+    private const ALLOWED_MIME_TYPES = [
```

Comment:
Can this list be configurable from container configuration? Eg. here put a default list, but provide an ability to specify mime types via constructor arguments.